### ISSUE TITLE
i3d_baseline_8x8_IN_pretrain_300k.log

### ISSUE THREAD

**Issue by jetyingjia** [facebookresearch/video-nonlocal-net#10]
2018-02-17 16:11:35.414	[INFO: test_net.py:  204]: gt_labels 19761, sample_num 19761
2018-02-17 16:11:39.536	[INFO: test_net.py:  249]: Num of empty videos: 0
2018-02-17 16:11:39.536	[INFO: test_net.py:  250]: Num of corrupted videos: 0
2018-02-17 16:11:39.536	[INFO: test_net.py:  251]: Max num of clips in a video: 10
2018-02-17 16:11:39.536	[INFO: test_net.py:  252]: Min num of clips in a video: 10
2018-02-17 16:11:39.536	[INFO: test_net.py:  256]: Clip1 accuracy: 64.66 percent (12777/19761)
2018-02-17 16:11:39.536	[INFO: test_net.py:  260]: Clip accuracy: 64.43 percent (127330/197610)
2018-02-17 16:11:39.995	[INFO: test_net.py:  283]: --------------------------------------------------------------------------------
2018-02-17 16:11:39.995	[INFO: test_net.py:  284]: top-1 accuracy: 71.89 percent
2018-02-17 16:11:39.995	[INFO: test_net.py:  285]: top-5 accuracy: 90.29 percent
2018-02-17 16:11:39.996	[INFO: test_net.py:  286]: --------------------------------------------------------------------------------
2018-02-17 16:18:15.690	[INFO: test_net.py:  182]: Temporary file saved to: ./results_probs.pkl
2018-02-17 16:18:16.287	[INFO: train_net.py:  253]: 10-clip spatial fcn testing finished

In this log, the performance is 71.89%, but in the web is  73.2%:

script | input frames | freeze bn? | 3D conv? | non-local? | top1 | top5 | model | logs
-- | -- | -- | -- | -- | -- | -- | -- | --
run_i3d_baseline_300k.sh | 8 | - | Yes | - | 73.2 | 90.8 | link | link

Is there something I missing or 73.2% is testing in another testing type?

**Comment by xiaolonw** [comment]
yes, some of the logs use testing with only center cropping, without fully convolutional testing... 
The reported number in README is based on fully convolutional testing

**Comment by jetyingjia** [comment]
the difference between In the log, 10-clip spatial fcn testing and fully convolution testing in the readme is?

**Comment by xiaolonw** [comment]
the log does not do fully convolutional testing, it is center cropping 224 * 224 from the frames. 
Fully convolution testing sees the whole image by: rescale the shorter side of the image to 256, and cropping 256 * 256 along the longer side for 3 times.
